builder/docs_builder.livecodescript

```diff
@@ -1987,6 +1987,8 @@ private command docsBuilderGetGuideFolders pEdition, @xFolders
       put RepoBusinessDocsFolder() & slash & "guides" into tBusinessGuides
       addFolderToListIfExists tBusinessGuides, false, xFolders
    end if
+
+   addFolderToListIfExists builderBuiltGuidesFolder(), false, xFolders
```

Comment:
Should the second parameter here be `true`, i.e. error if there is no folder, seeing as the release notes should always be there?